### PR TITLE
Create krvishal.json

### PR DESCRIPTION
**Pull request by imshvishal**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [] The website is reachable.
- [ ] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/krvishal.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/thegeekdex",
+    "owner": {
+        "username": "thegeekdex",
+        "email": "",
+        "discord": "Vishal S.#6993"
```
Comment (WilliamDavidHarrison) [inline domains/krvishal.json:7]: This is optional but recommended:

It is best to include your Discord ID, just incase your tag changes.
Like this: `"note": "My Discord ID is: 1234567890"`

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: domains/krvishal.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    },
+    "record": {
+         "A": ["3.108.152.235"]
```
Comment (WilliamDavidHarrison) [inline domains/krvishal.json:10]: Please complete your website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/krvishal.json (hunk 2 above)
Comment (TheGeekDex) [reply domains/krvishal.json:10]: its complete, but can't be proxified without domain.

Comment (WilliamDavidHarrison) [reply domains/krvishal.json:10]: @TheGeekDex 
image

Comment (TheGeekDex) [reply domains/krvishal.json:10]: nothing is in construction, just need to update that in database. you can see, everything is working fine

**Review by andrewstech — APPROVED** [review]

**Pull request merged**